rawdata/src/main/java/com/griddynamics/jagger/rawdata/util/KafkaTopicCreator.java

```diff
@@ -0,0 +1,47 @@
+package com.griddynamics.jagger.rawdata.util;
+
+import com.griddynamics.jagger.rawdata.receiver.RawDataReceiver;
+import kafka.admin.AdminUtils;
+import kafka.utils.ZKStringSerializer$;
+import kafka.utils.ZkUtils;
+import org.I0Itec.zkclient.ZkClient;
+import org.I0Itec.zkclient.ZkConnection;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Properties;
+
+@SuppressWarnings("unused")
+public class KafkaTopicCreator {
```

Comment:
Thank you, I know. But we will disable this feature on the broker, that's why we need this class.